docs/platforms/php/index.mdx

```diff
@@ -25,28 +25,73 @@ Sentry captures data by using an SDK within your application’s runtime. These
 
 Install the SDK using [Composer](https://getcomposer.org/).
 
+
+<OnboardingOptionsButtons
+  options={[
+    {
+      id: 'error-monitoring',
+      name: 'Error Monitoring',
+      disabled: true,
+      tooltip: "Let's admit it, we all have errors.",
+    },
+    {
+      id: 'performance',
+      name: 'Performance Monitoring',
+      tooltip:
+        'Automatic performance issue detection across services and context on who is impacted,' +
```

Comment:
The tooltips / names will repeat on every page we use the component. Can we move that into the component itself, so tooltips etc must only be maintained in a single place? Also stuff like the dependency is always the same across SDKs.